code/modules/clothing/under/accessories.dm

```diff
@@ -360,3 +360,21 @@
 	above_suit = TRUE
 	armor = list("melee" = 5, "bullet" = 5, "laser" = 5, "energy" = 5, "bomb" = 20, "bio" = 20, "rad" = 5, "fire" = 0, "acid" = 25)
 	attachment_slot = GROIN
+
+//////////////
+//Ooh shiny!//
+//////////////
+
+/obj/item/clothing/accessory/sing_necklace
+	name = "Singularity Shard Necklace"
+	desc = "The gem mounted inside seems to glow with an unearthly, pulsing light. It is cold to the touch."
+	icon_state = "sing_necklace"
+	item_color = "sing_necklace"
+	above_suit = TRUE
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = -5, "fire" = 0, "acid" = 0) //It IS radioactive after all. Watch me get yelled at for powergaming because I'm making this my """donator""" item - Mqiib
```

Comment:
```suggestion
	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0) //It IS radioactive after all. Watch me get yelled at for powergaming because I'm making this my """donator""" item - Mqiib
```

Reply:
You're no fun. It's NEGATIVE armor, not actually helpful.

Reply:
and if you're going to delete the minor negative effect then get rid of the comment about it too.